msal-python-conceptual/advanced/macos-broker.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Using MSAL Python with Authentication Brokers on macOS
+description: "Using authentication brokers on macOS enables you to simplify how your users authenticate with Microsoft Entra ID from your application, as well as take advantage of advanced functionality such as token binding, protecting any issued tokens from exfiltration and misuse."
```

Comment:
A few nit picks:
1. We only have a single broker on Mac, so should be using singular
2. Token binding isn't public yet on Mac and it only protects refresh tokens for now, I suggest rewording last sentence to more clearly represent the benefit